app/components/schedule/Fullitem.vue

```diff
@@ -0,0 +1,223 @@
+          {{ removeSeconds(show.end) }}
+        </div>
+        <div class="onairshow">
+          <span class="text-red-600">●</span>
+          On Air
+        </div>
+      </div>
+      <div class="flex justify-between w-full schedule-infos">
+        <div v-if="showAirCheck(show.name)" class="flex">
+          <div class="mr-4 onair-image">
+            <img :src="onAirImage" :alt="show.name">
+          </div>
+          <div class="onair-infos">
+            <NuxtLink :to="'/shows/' + show.archive_lahmastore_base_url" class="block mb-2">
+              <b>{{ show.name }} {{ showAirCheck(show.name) && ' | ' + streamEpisodeTitle }} </b>
+            </NuxtLink>
+            <div class="mb-2 text-sm onair-meta">
+              {{ showFrequency(show.frequency, show.week) }} |
+              Language: <span v-sanitize.nothing="getLanguageGraph(show.language)" class="language" />
+            </div>
+            <div class="text-sm description">
+              {{ onAirDescription }}
+            </div>
+          </div>
+        </div>
+        <div v-else-if="opened" class="flex">
+          <div class="mr-4 onair-image">
+            <NuxtLink :to="latestEpisodeLink" class="block mb-2">
+              <img :src="latestEpisodeImage" :alt="show.name">
+            </NuxtLink>
+          </div>
+          <div class="onair-infos">
+            <NuxtLink :to="latestEpisodeLink" class="block mb-2">
+              <b>{{ latestEpisodeTitle }} </b>
+            </NuxtLink>
+            <div class="text-sm description">
+              {{ onAirDescription }}
+            </div>
+          </div>
+        </div>
+        <div v-else>
+          <NuxtLink :to="'/shows/' + show.archive_lahmastore_base_url" class="block">
+            <b>{{ show.name }}</b>
+          </NuxtLink>
+        </div>
+        <div v-if="!showAirCheck(show.name)">
+          <a href="#" @click.prevent="opened = !opened">
+            <i v-if="opened" class="fa fa-chevron-up" aria-hidden="true" />
+            <i v-else class="fa fa-chevron-down" aria-hidden="true" />
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+  </div>
+</template>
+
+<script>
+import { mediaServerURL } from '~/constants'
+
+export default {
+  props: {
+    show: {
+      type: Object,
+      required: true
+    },
+    nowPlaying: {
+      type: Object,
+      required: true
+    }
+  },
+  data () {
+    return {
+      opened: false
+    }
+  },
+  computed: {
+    isTouchEnabled () {
+      return ('ontouchstart' in window) || (navigator.maxTouchPoints > 0) || (navigator.msMaxTouchPoints > 0)
+    },
+    streamShowTitle () {
+      if (!this.nowPlaying.now_playing) {
+        return false
+      } else if (this.nowPlaying?.live?.is_live) {
+        return this.nowPlaying?.live?.streamer_name
+      } else {
+        return this.nowPlaying?.now_playing?.song.artist
+      }
+    },
+    streamEpisodeTitle () {
```

Comment:
These are some wacky things!! 

If we use these in other places should we create a global helper component where these can be written and then imported anywhere?

Reply:
Yes, it would be possible to have a global mixin, but we use this at two places in my knowledge, and it is quite stream centered